feat: add build url on pact publish

konalegi opened the pull request:
Add `buildUrl` parameter for the pact publication function. Follow-up for https://github.com/pact-foundation/pact-js-core/issues/352

TimothyJones commented:
Thanks for this!

Two small issues:

1) The commit message here will appear in the release notes - However, this message currently isn't suitable for the release notes as it's a bit cryptic. (see our instructions here: https://github.com/pact-foundation/pact-js-core/blob/master/CONTRIBUTING.md#release-notes for guidelines). 

Could you choose one of:

* Force push (or otherwise resubmit) with a release-notes appropriate message (if you're not sure what to put, I think something like `feat: Add `buildUrl` option to Verifier API`)
* Let us squash merge it with a new message (if we do this, you don't get appropriately marked as a contributor, I think)

2) Could you add the new option to the documentation in the readme please?

TimothyJones commented:
Also, it's currently failing the prettier format check step. You can fix this with `npm run format:fix` and committing the result.

konalegi commented:
@TimothyJones thanks for your reply. 
* Added documentation
* Fixed prettier
* Updated commit message but feel free to update it before merging.

Thanks!

TimothyJones commented:
Thanks very much! I'll release this now

TimothyJones commented:
This is now released in `pact-core@13.4.0` and `pact-node@10.16.0`

Thanks again for the PR! Welcome to the list of pact-js contributors :)
